docling_ibm_models/code_formula_model/utils/constants.py

```diff
@@ -0,0 +1,6 @@
+DEFAULT_IMAGE_PATCH_TOKEN = "<imgpad>"
```

Comment:
All these constants are used in only one short scope of the predictor, please eliminate this unit and move the constants into the predictor module.